client/src/Components/NotFound/NotFound.js

```diff
@@ -1,16 +1,23 @@
 import React from 'react';
 import './NotFound.css';
+import PropTypes from 'prop-types';
 
-const NotFound = () => {
+const NotFound = ({err}) => {
+  const message = err?.message.includes('5') ? "There has been a server error." : "This page either moved or doesn't exist.";
```

Comment:
I'm super curious to know how to use a regex to check for a 500 level error but I also am exhausted